Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Generic_v2.yml

```diff
@@ -261,30 +627,42 @@ inputs:
     Possible values: True / False.
     The default value is false.
   playbookInputQuery:
+- key: ExecutedFromParent
+  value:
+    simple: "True"
+  required: false
+  description: |-
+    Whether to execute common logic, like the classification of IP addresses to ranges and resolving, in the the main (IP Enrichment - Generic v2) enrichment playbook, instead of in the sub-playbooks.
+    Possible values are: True, False.
+    
+    Setting this to True will execute the relevant commands in the main playbook instead of executing them in both sub-playbooks.
+
+    Set this to True in the parent playbook if you are using the parent playbook, as opposed to using the sub-playbooks directly in your playbooks, as this will improve the performance of the playbook and reduce the overfall size of the incident.
```

Comment:
```suggestion
    Set this to false in the parent playbook if you are using the parent playbook, as opposed to using the sub-playbooks directly in your playbooks, as this will improve the performance of the playbook and reduce the overfall size of the incident.
```